dl-snapchat.html

```diff
@@ -3,18 +3,51 @@
 layout: blank
 title: "Vibe Check"
 ---
+
 <div class="page-vibe-check">
-  <div class="logo">
-      <img src="/assets/images/favicon/android-chrome-192x192.png"/>
-  </div>
-  <div class="body-text">
-    Turn&nbsp;your Snapchat stories&nbsp;into dreams.
-  </div>
-  <div class="button">
-    <a  href="https://thetrash.app.link/36xcKYfZr7" target="_blank">
-      <img class="off" src="/assets/images/appstorebutton_rainbow.svg"/>
-      <!-- <img class="on" src="/assets/images/appstorebutton_white.svg" /> -->
-    </a>
-  </div>
+	<div class="content">
+		<svg
+			width="62"
+			height="62"
+			viewBox="0 0 82 82"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<title>TRASH App</title>
+			<defs>
+				<linearGradient x1="100%" y1="0" x2="0" y2="0" id="a">
+					<stop stop-color="#BB68FD" offset="2.1%" />
+					<stop stop-color="#9B85FF" offset="17.2%" />
+					<stop stop-color="#77CCE6" offset="31.8%" />
+					<stop stop-color="#8EFFA7" offset="46.9%" />
+					<stop stop-color="#F1FF6B" offset="62.5%" />
+					<stop stop-color="#F0C559" offset="78.6%" />
+					<stop stop-color="#FF5D9C" offset="94.8%" />
+				</linearGradient>
+			</defs>
+			<g fill="none">
+				<path
+					d="M58.267.333L23.265.334l-.95.008c-6.332.084-9.383.741-12.557 2.439A16.78 16.78 0 0 0 2.78 9.758C.966 13.152.333 16.425.333 23.733l.001 35.002.008.95c.084 6.332.741 9.383 2.439 12.557a16.78 16.78 0 0 0 6.977 6.977c3.394 1.815 6.667 2.448 13.975 2.448h34.534c7.308 0 10.581-.633 13.975-2.448a16.78 16.78 0 0 0 6.977-6.977c1.815-3.394 2.448-6.667 2.448-13.975V23.733c0-7.308-.633-10.581-2.448-13.975a16.78 16.78 0 0 0-6.977-6.977C68.848.966 65.575.333 58.267.333zm-35 1.335l35-.001c7.106 0 10.172.592 13.347 2.29a15.447 15.447 0 0 1 6.43 6.43l.224.432c1.526 3.029 2.065 6.13 2.065 12.914v34.534c0 7.106-.592 10.172-2.29 13.347a15.447 15.447 0 0 1-6.43 6.43l-.432.224c-3.029 1.526-6.13 2.065-12.914 2.065H23.733c-7.106 0-10.172-.592-13.347-2.29a15.447 15.447 0 0 1-6.43-6.43l-.23-.444c-1.49-2.961-2.03-5.978-2.058-12.437l-.001-34.999c0-7.106.592-10.172 2.29-13.347a15.447 15.447 0 0 1 6.43-6.43l.444-.23c2.961-1.49 5.978-2.03 12.437-2.058z"
+					fill="#FFF"
+				/>
+				<path
+					fill="#FFF"
+					d="M61.877 38.061H45.789v28.183h-9.772V38.061H19.93v-7.818h41.947z"
+				/>
+				<path
+					fill="url(#a)"
+					d="M1.015 13.292L0 5.074 41.572 0l1.015 8.219z"
+					transform="translate(19 15)"
+				/>
+			</g>
+		</svg>
+		<p class="body-text">
+			Turn your Snapchat
+			<br />
+			stories into dreams.
+		</p>
+	</div>
+	<a href="https://thetrash.app.link/36xcKYfZr7" target="_blank">
+		<img class="off" src="/assets/images/appstorebutton_rainbow.svg" />
+		<!-- <img class="on" src="/assets/images/appstorebutton_white.svg" /> -->
```

Comment:
maybe this can be enabled now that the bg is black again. it was looking awkward on a whilte bg.

Reply:
sure, can do

Reply:
at armand's request i'm gonna change these out for a button with "download trash"